src/view/components/controlledCombo.jsx

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import React from "react";
+import ComboBox from "@react/react-spectrum/ComboBox";
+
+// Use this component instead of a regular ComboBox to avoid the bug where you have to click
+// twice to select an option
```

Comment:
I think it would be good to add the issue number to the comment so that we can remove this module when it's fixed.

Reply:
How do I reproduce the problem? I tested with the following code and didn't have to click twice to select an option:

```
export default class ControlledCombo extends React.Component {
  // state = {
  //   showMenu: false
  // };

  render() {
    const {
      props,
      // state: { showMenu }
    } = this;
    return (
      <ComboBox
        // showMenu={showMenu}
        // onMenuToggle={menuState => this.setState({ showMenu: menuState })}
        {...props}
      />
    );
  }
}
```

Reply:
When I used the ComboBox directly, I could reproduce the double clicking bug.  When I looked at the react-spectrum storybook code, I noticed it created a ControlledCombo control, so I figured the showMenu state was necessary.

Reply:
I just tried again using the ComboBox directly and I'm not seeing any issues. Am I missing something?

My guess with the react-spectrum storybook is that they want to demonstrate that it's possible to control the menu state, but it probably isn't necessary.

Reply:
I tried it again and it worked without the controlledCombo.  Thanks for looking into this.  I thought it was odd to have to do that.